noxfile.py

```diff
@@ -0,0 +1,69 @@
+from shutil import rmtree
+from textwrap import dedent
+from pathlib import Path
+import nox
+
+ROOT = Path(__file__).parent
+STYLEGUIDE = ROOT / "styleguide"
+SRC_DIRECTORY = STYLEGUIDE
+BUILD_DIRECTORY = STYLEGUIDE / "_build"
+
+nox.options.sessions = ['build', 'open']
+
+
+@nox.session(python=None)
+def build(session):
+    """Build the styleguide."""
+    session.run(
+        "sphinx-build",
```

Comment:
maybe introduce directly the sphinx github pages generator

Reply:
I would say can be added later if needed, especially because I don't think there is a need to have different versions of the styleguide. The styleguide is in flux and the current master should reflect the current state of the art.

Reply:
Or where you want to go. And, it is clear that we want to deploy the documentation on GitHub pages and this with the Sphinx GitHub pages Generator

Reply:
For me personally GH pages would be preferable, but if this is not ideal or disallowed, I am open to suggestions.
@redcatbear from company policy point of view would GH-Pages be ok?

Reply:
@Nicoretti , yes it is. @MarleneKress79789 and @tkilias worked on 
https://github.com/exasol/sphinx-github-pages-generator
which simplifies the generation and deployment of GH pages.

I believe Torstens suggestion is to use this tool also here.

Reply:
@tomuben thanks for clarifying. 
@tkilias , @tomuben if I have seen correctly, currently there is no "release" of this package yet, neither on `pipy` nor on `GH`. So personally I would rather wait for `sphinx-github-pages-generator` to provide package for the following reasonds:

* By initially having "pure" sphinx then using the generator we will validate that the results is the "same" (expected one)
* By having using it via the package, we can check if the package is correct and working
* It forces the project to do packaging (otherwise it increases the likelihood that packaging gets neglected in the long run)

Reply:
@Nicoretti you can see, making this release as your next task regarding the styleguide, ;-)